### DISCUSSION TITLE
How do I update a file I attached to an issue?

### DISCUSSION THREAD

**Discussion by Rod-at-DOH** [community/community#106814] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Earlier today I created a new issue in one of my repos. I saw that I could attach a file to an issue. The users had provided me with a Word document listing all of the changes they wanted. So, to save time I decided to attach the Word document to the new issue.

Then a few hours later the users gave me an updated Word document with new information. ☹️ 

So, how do I update the file I attached to the issue when I created it?

**Comment by gil-son** [comment]
You can use that button to attach one or more files:
![github06](https://github.com/community/community/assets/72712095/83ce9ae1-e995-4c89-9287-3d2ee86c76db)

**Reply by Rod-at-DOH** [reply]
Do I do that to the original post I used to create the issue? Or do a comment on the original post?

**Reply by gil-son** [reply, marked as answer]
Consider to open a Ticket about that issue

Acces the [Github Support](https://support.github.com/)

Click on My Ticket :

![github04](https://github.com/community/community/assets/72712095/096b3477-cee6-4a19-9f1b-6fb2219b51a1)

and open a New Ticket:

![github05](https://github.com/community/community/assets/72712095/7413fc55-875a-498c-93eb-5c54d5ffc8d6)


God bless you!
